### ISSUE TITLE
link publico para listas associados  pagantes

### ISSUE THREAD

**Issue by cadu-leite** [pythonbrasil/associados#128]
existe uma url para listar os membros.  

    http://associados.python.org.br/members/list/

1 - poderia estar mais aparente
2 - só mostrar os ATIVOS (que pagaram nos ultimos 12 meses)

**Comment by ocarneiro** [comment]
Concordo. E podia ter link direto para cada letra do alfabeto. Aí a pessoa pode colocar no currículo: "membro da Apyb", com um link da página onde ele aparece, caso um potencial empregador queira conferir.

**Comment by filipecifali** [comment]
Por mais aparente vocês dizem tipo um slide show na página inicial com membros aleatórios? 

Desenhei isso daqui bem simples porque sou ruim com essas coisas:

![drawing](https://cloud.githubusercontent.com/assets/975041/18435864/f0577ad4-78cb-11e6-9c74-aa2152439251.png)

O que vocês acham

**Comment by filipecifali** [comment]
Para a página da lista de membros: 

![drawing 1](https://cloud.githubusercontent.com/assets/975041/18436082/e49d1874-78cc-11e6-96e0-b1e141bd19c7.png)

Imagino algo como isso, o que acham?

**Comment by igr-santos** [comment]
@filipecifali curti muito essa sua última tela
